models/marketing/intermediate/int_hubspot__contact_merge_adjust.sql

```diff
@@ -1,21 +1,25 @@
-{{ config(enabled=fivetran_utils.enabled_vars(['hubspot_marketing_enabled', 'hubspot_contact_enabled', 'hubspot_contact_merge_audit_enabled'])) }}
+{{ config(enabled=fivetran_utils.enabled_vars(['hubspot_marketing_enabled', 'hubspot_contact_enabled'])) }}
 
 with contacts as (
 
     select *
     from {{ var('contact') }}
-
 ), contact_merge_audit as (
+{% if var('hubspot_contact_merge_audit_enabled', false) %}
```

Comment:
Discussed during standup. Since this is very explicitly relating to the contact_merge_audit table and not the actual merging, it is appropriate to keep the variable the same name.